boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -89,9 +89,24 @@
 		mode-gpios  = <&gpio1 12 GPIO_ACTIVE_HIGH>;
 		pdn-gpios   = <&gpio1 10 GPIO_ACTIVE_HIGH>;
 		tx-en-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		gpio-fwd = <&gpio_fwd>;
 		spi-if = <&nrf_radio_fem_spi>;
 	};
 
+	gpio_fwd: nrf-gpio-forwarder {
+		compatible = "nordic,nrf-gpio-forwarder";
+		status = "okay";
+		fem-gpio-if {
+			gpios = <&gpio1 11 GPIO_ACTIVE_HIGH>,
```

Comment:
I understand GPIO flags from these nodes are ignored. I would change their values to 0 and state in the binding file that GPIO flags are ignored, and must be set in the matching binding in the forwarded core device tree nodes using these pins.